src/__tests__/Grid.Col.test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-console */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { shallow } from 'enzyme';
+
+import { Col } from '../Grid';
+
+const MockComponent = props => <div css={props.styling} />;
+MockComponent.propTypes = {
+  styling: PropTypes.string.isRequired,
+};
+
+let workingConsoleError;
+
+/*
+ * Before/After each test we temporarily assign console.error to a mock function,
+ * so that we don't get ugly red warnings in the terminal when running tests!
+ * This allows us to check that we would be throwing errors correctly in real life,
+ * such as if someone passed an incorrect argument to the Col method!
+ */
+beforeEach(() => {
+  // NB. **ALWAYS** remember to reset the mocked console messages back to their original state!!
+  workingConsoleError = console.error;
+  console.error = jest.fn();
+});
+
+afterEach(() => {
+  console.error = workingConsoleError;
+});
+
+describe('Grid : Col(n) method', () => {
+  it('should render via Emotion as CSS when passed as a className to a React component', () => {
```

Comment:
Depending on action taken elsewhere, this test title should either refer explicitly to a `css` prop, or perhaps `should be passable as a prop`. `className` isn't how the class gets added anymore.

Reply:
You're right, I think I overlooked the need to update the test descriptions to be accurate to what's really happening.